src/index.js

```diff
@@ -0,0 +1,39 @@
+import './style.css';
+import ListTasks from './modules/ListTasks.js';
+
+/* List Task */
+const listTask = new ListTasks();
```

Comment:
According to the project requirements, by default, your app should contain a hardcoded array with some simple to-do tasks containing three keys. Then the tasks are used to populate a HTML list item. Please update your application to include this feature.